setup-gcloud/README.md

```diff
@@ -0,0 +1,52 @@
+<!--
+ Copyright 2019 Google LLC
+
+ Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in
+ compliance with the License. You may obtain a copy of the License at
+
+        https://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software distributed under the License
+ is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ implied. See the License for the specific language governing permissions and limitations under the
+ License.
+-->
+
+# setup-gcloud
+
+This action configures the [Google Cloud SDK](https://cloud.google.com/sdk/) in the environment for use in actions.
+
+It does the following:
+
+1. Downloads a version of the [Google Cloud SDK](https://cloud.google.com/sdk/) according to the specified `version` input, as well as the environment OS and architecture.
```

Comment:
It's helpful to mention the Cloud SDK == gcloud

Reply:
Done.